omod/pom.xml

```diff
@@ -18,6 +18,15 @@
         <gem.path>${basedir}/.rubygems</gem.path>
     </properties>
 
+    <!-- Repository for patient search criteria -->
+    <repositories>
+		<repository>
+		    <id>jitpack.io</id>
```

Comment:
any idea why u wanna publish these on jitpack?

Reply:
@kaweesi sir, I added this repository in PR so you can use the patient search criteria module(which is not merge yet). Once you merge patient search criteria module PR(https://github.com/openmrs/openmrs-module-patientsearchcriteria/pull/1)  and create its remote repository, I will replace this jitpack with that repository. Another thing is that the PR of patient search criteria also consists of a jitpack repository of openmrs-core(https://github.com/openmrs/openmrs-core/pull/2980).

Overall, there are dependencies between the PRs as I explained in the talk.
please 1st merge openmrs core PR(https://github.com/openmrs/openmrs-core/pull/2980).
Then merge patient search criteria module PR(https://github.com/openmrs/openmrs-module-patientsearchcriteria/pull/1) and create a remote repository of that module and at last, after creating a repository of patient search criteria module I just need to replace jitpack repository with patient search criteria repository